latest/src/startup/entrypoint.sh

```diff
@@ -3,7 +3,32 @@ export USER=headless
 
 /dockerstartup/vnc_startup.sh &
 
-/opt/squish.run unattended=1 ide=0 targetdir=${HOME}/squish licensekey=$LICENSEKEY
+#maximum time to wait for licenses (before installation of squish + before running tests)
+runtime="30 minute"
+endtime=$(date -ud "$runtime" +%s)
+
+result=1
+echo "installing squish"
+
+# retry installing squish if there is an issue to connect the license server
+while [[ $result -ne 0 ]]
+do
+  if [[ $(date -u +%s) -gt $endtime ]]
+  then
+    echo "timeout waiting for license server"
+    exit 1
+  fi
+
+  /opt/squish.run unattended=1 ide=0 targetdir=${HOME}/squish licensekey=$LICENSEKEY
+  result=$?
+
+  if [[ $result -ne 0 ]]
+  then
+    echo "sleeping waiting for license server"
+    sleep $((1 + $RANDOM % 30))
```

Comment:
Note: `$RANDOM` returns a pseudorandom integer in the range 0 - 32767. 1 + mod 30 gives an integer in the range 1 to 30 inclusive.